<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpaqViewer</class>
 <widget class="QWidget" name="OpaqViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="stationLabel">
           <property name="text">
            <string>Station:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="stationComboBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="pollutantLabel">
           <property name="text">
            <string>Pollutant:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="pollutantComboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="aggregationLabel">
           <property name="text">
            <string>Aggregation:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="aggregationComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="basetimeLabel">
           <property name="text">
            <string>Basetime:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDateEdit" name="basetimeDateEdit">
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="simulateButton">
         <property name="text">
          <string>Simulate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="OPAQ::ResultsView" name="resultsView" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OPAQ::ResultsView</class>
   <extends>QWidget</extends>
   <header>resultsview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
